### DISCUSSION TITLE
What's the difference between "Toggle Translate Page" and "Toggle Translate the Main Page"??

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [immersive-translate/immersive-translate#992] [Help] [answered]
### 概述 | Summary

I don't understand the difference between "Toggle Translate Page" and "Toggle Translate the Main Page", I see no difference when using it.

"Toggle Translate the Whole Page" --> translate the title, the menu etc...
"Toggle Translate the Main Page" --> translate everything below the menu
"Toggle Translate Page" -->  same

**Comment by theowenyoung** [comment, marked as answer]
"Toggle Translate Page" -->  this trigger is depended by your current translation mode, 

for example, if you are in "Translate the Whole Page" mode now, it will translate the whole page, 

if you are in "Toggle Translate the Main Page", it wil translate the main content only.

but for the following triggers will always  Translate the Whole Page or  Translate the Main Page

"Toggle Translate the Whole Page" --> translate the title, the menu etc... 
"Toggle Translate the Main Page" --> translate everything below the menu
